tests/test_cli/test_generate/test_generate.py

```diff
@@ -21,7 +21,7 @@
 
 from click import ClickException
 
-from aea.cli.generate import _generate_item
+from aea.cli.generate import protocol
```

Comment:
That's a problematic method name. Tells where little what it does. I preferred `_generate_item/protocol`